Tests/APITesters/CustomEntitlementComputationSwiftAPITester/SwiftAPITester/PurchasesAPI.swift

```diff
@@ -19,12 +19,12 @@ func checkPurchasesAPI() {
     let purch = checkConfigure()!
 
     // initializers
-    let finishTransactions: Bool = purch.finishTransactions
```

Comment:
Should we keep this deprecated API somewhere still since it's still public? 🤔

Reply:
yeah, we should keep it until it's officially removed from the API